springboard/templates/base.jinja2

```diff
@@ -14,17 +14,16 @@
     <div id="header">
         <div class="lang">
             <div>
-                {% for code, name in languages %}
-                <span>
-                    <a href="/locale/?language={{code}}">{{name}}</a>
-                    {% if not forloop.last %}
-                        <span> |</span>
-                    {% endif %}
-                </span>
+                {% for lang_code in view.get_display_languages() %}
```

Comment:
This can now be `display_languages` because it's a variable in the default template context.